src/placement/pl_map_common.c

```diff
@@ -386,21 +386,22 @@ pl_map_extend(struct pl_obj_layout *layout, d_list_t *extended_list,
 	      bool is_pool_adding)
```

Comment:
not a problem of this patch, but since this patch removed the usage of is_pool_adding, looks better to remove it from parameter list.